src/LibraryMangement/library.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import { data } from './data';
+// import './style.css';
+const Library = () => {
+
+    const [libraryData, setLibraryData] = useState(data);
+    const [filterData, setfilterData] = useState(data);
+    const [searchdata, setsearchdata] = useState('');
+
```

Comment:
please explain  all these states one by one, why do we need it and what data that is stores ?

Reply:
```     const [libraryData, setLibraryData] = useState(data);
    const [filterData, setfilterData] = useState(data);
    const [searchdata, setsearchdata] = useState('');

````